demo/src/App.js

```diff
@@ -585,6 +601,7 @@ class App extends Component {
             </div>
             <div ref={tc => (this.treeContainer = tc)} className="tree-container">
               <Tree
+                interactiveNode
```

Comment:
See comment on `src/Tree/types.ts`

```suggestion
                hasInteractiveNodes
```